pkgs/development/python-modules/magic-wormhole-mailbox-server/default.nix

```diff
@@ -48,6 +49,12 @@ buildPythonPackage rec {
     twisted
   ];
 
+  # Fails in Darwin's sandbox
```

Comment:
This is disabling some tests that are failing in Darwin's sandbox. I cannot use `disableTests` nor `disableTestPaths` because it is using `twisted.trial` framework and we don't have a test hook on it (`pytestCheckHook` is for `pytest` framework).

Let me remove my change and run the build again to provide a log.

Reply:
I got inspired to do it this way from the `python3Packages.twisted` nix configuration.
https://github.com/NixOS/nixpkgs/blob/9870f334cbf2f822d273eba1cd05ef061d21d46a/pkgs/development/python-modules/twisted/default.nix#L104-L151
Since it's also using the `twisted.trial` framework.

Reply:
Here attached the logs.
[python3.10-magic-wormhole-mailbox-server-0.4.1.txt](https://github.com/NixOS/nixpkgs/files/12498826/python3.10-magic-wormhole-mailbox-server-0.4.1.txt)
most of the errors are something like:
`twisted.internet.error.CannotListenError: Couldn't listen on 127.0.0.1:0: [Errno 1] Operation not permitted.`